### PR TITLE
Show error on hover in the transactions screen

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Show error on hover in the transactions screen](https://trello.com/c/naj42qoi/319-show-error-on-hover-in-the-transactions-screen)

File: new-lamassu-admin/src/pages/Transactions/helper.js [hunk 1]
```diff
@@ -21,11 +18,7 @@ const getCashInStatus = it => {
 }
 
 const getCashInStatusDetails = it => {
```
Comment (liordino) [inline new-lamassu-admin/src/pages/Transactions/helper.js]: The getCashInStatusDetails and getCashOutStatusDetails functions have the same body. Can turn these into a single function

File: new-lamassu-admin/src/components/Tooltip.js [hunk 2]
```diff
@@ -51,4 +51,35 @@ const Tooltip = memo(({ children, width, Icon = HelpIcon }) => {
   )
 })
 
-export default Tooltip
+const HoverableTooltip = memo(({ parentElements, children, width }) => {
```
Comment (liordino) [inline new-lamassu-admin/src/components/Tooltip.js:66]: There's a good chunk of repeated code within Tooltip and HoverableTooltip. Creating another component used by both of them with the common parts or parameterizing the behaviour could clean it up.

**Review by liordino — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/components/Tooltip.js (hunk 2 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/components/Tooltip.js:66]: I agree, My solution was creating a hook which can be used by other Tooltip-like components, regarding the popper handling

File: new-lamassu-admin/src/pages/Transactions/helper.js (hunk 1 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Transactions/helper.js]: Although the bodies are the same, I believe it is important to keep them separated for readability's sake and as they serve different directions of transactions, their logic should be kept separated, despite being equal, for now.

Tell me what you think about this and if my argument has changed your mind about this

Comment (liordino) [reply new-lamassu-admin/src/pages/Transactions/helper.js]: I think that we can keep a single one while the logic is one and separate when they become different things in fact, if this ever occurs (may as well be never). By doing this we avoid, for example, updating one function while forgeting to change the other, which can cause some hard to track bugs. I agree that some readability may be lost in the process (even though while having only one behaviour keeping a single function is simpler and represents the current truth), but we avoid adding unnecessary complexity to the code, which can add up fast.

**Review by liordino — CHANGES_REQUESTED** [review]
Didn't noticed that before, but needs to sign the commits.

**Comment by chaotixkilla** [comment]
@liordino All 4 commits are showing up as signed to me, can you please double-check?

**Comment by liordino** [comment]
> @liordino All 4 commits are showing up as signed to me, can you please double-check?

@chaotixkilla you're right, my mistake. It shows the signature as invalid in sublime merge for me, and I trusted it, but it's indeed signed.

**Review by liordino — APPROVED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**